src/components/Clock.tsx

```diff
@@ -0,0 +1,59 @@
+import { Component } from 'react';
+
+interface Props {
+  name: string;
+}
+
+interface State {
+  today: Date;
+  id: number;
+}
+
+export class Clock extends Component<Props, State> {
+  state = {
+    today: new Date(),
+    id: 0,
```

Comment:
what's the purpose of this property in state?

Reply:
you didn't delete this property
what's the reason of adding `id` in state?

Reply:
there is no reason, sorry, I just didn't notice this comment before